projects/hip-tests/catch/packaging/CMakeLists.txt

```diff
@@ -77,72 +75,66 @@ foreach(CTEST_INCLUDE_FILE ${INSTALL_CTEST_INCLUDE_FILES})
+  # Remove once mlsejenkins is deprecated
+  if(HIPTESTS_INTERNAL_BUILD)
+    set(INSTALL_DIR .)
   endif()
-
-  execute_process( COMMAND rpm --eval %{?dist}
-                  RESULT_VARIABLE PROC_RESULT
-                  OUTPUT_VARIABLE EVAL_RESULT
-                  OUTPUT_STRIP_TRAILING_WHITESPACE )
-  # Add os distribution tag to rpm package name . For deb package its set from build env
-  if ( PROC_RESULT EQUAL "0" AND NOT EVAL_RESULT STREQUAL "" )
-    string(APPEND CPACK_RPM_PACKAGE_RELEASE "%{?dist}")
+  if(NOT DEFINED CPACK_INSTALL_PREFIX)
+    set(CPACK_INSTALL_PREFIX .)
   endif()
-  set(CPACK_SOURCE_GENERATOR "TGZ")
-  # Install license file
-  set ( CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/../LICENSE.md" )
-  install(FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION ${INSTALL_DIR})
-  set(CPACK_RPM_PACKAGE_LICENSE "MIT")
-else()
-  # windows packaging
-  set(INSTALL_DIR ${CMAKE_INSTALL_PREFIX})
-  set(CPACK_INSTALL_PREFIX "")
-  set(CPACK_SYSTEM_NAME "")
   set(CPACK_GENERATOR "ZIP" CACHE STRING "Windows package types for catch tests")
-  set(CPACK_TEST_ZIP "ON")
-  set(CPACK_ZIP_TEST_PACKAGE_NAME "catch")
-  set(CPACK_PKG_DIRS
-      ${CMAKE_BINARY_DIR}/_CPack_Packages/${CPACK_GENERATOR}/${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}-${CPACK_SYSTEM_NAME})
-  set(CPACK_INSTALL_PREFIX ${CPACK_PKG_DIRS})
-  if(${CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT})
-    set(INSTALL_DIR ${CPACK_PKG_DIRS})
-  else()
-    set(INSTALL_DIR ${CMAKE_INSTALL_PREFIX})
-  endif()
 endif()
+set(CPACK_BINARY_DEB "ON")
+set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
+set(CPACK_DEBIAN_PACKAGE_PROVIDES "catch")
+set(CPACK_BINARY_RPM "ON")
+set(CPACK_RPM_FILE_NAME "RPM-DEFAULT")
+set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/opt")
+if (CPACK_PACKAGE_VERSION MATCHES "local" )
+  #If building locally default value will cause build failure
+  #DEBUG SYMBOL pacaking require SOURCE_DIR to be small
+  set(CPACK_RPM_BUILD_SOURCE_DIRS_PREFIX ${CPACK_INSTALL_PREFIX})
+endif()
+if (DEFINED ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
+  set(CPACK_DEBIAN_PACKAGE_RELEASE $ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
+else()
+  set(CPACK_DEBIAN_PACKAGE_RELEASE "local")
+endif()
+if(DEFINED ENV{CPACK_RPM_PACKAGE_RELEASE})
+  set(CPACK_RPM_PACKAGE_RELEASE $ENV{CPACK_RPM_PACKAGE_RELEASE})
+else()
+  set(CPACK_RPM_PACKAGE_RELEASE "local")
+endif()
+execute_process( COMMAND rpm --eval %{?dist}
+                RESULT_VARIABLE PROC_RESULT
+                OUTPUT_VARIABLE EVAL_RESULT
+                OUTPUT_STRIP_TRAILING_WHITESPACE )
+# Add os distribution tag to rpm package name . For deb package its set from build env
+if ( PROC_RESULT EQUAL "0" AND NOT EVAL_RESULT STREQUAL "" )
+  string(APPEND CPACK_RPM_PACKAGE_RELEASE "%{?dist}")
+endif()
+set(CPACK_SOURCE_GENERATOR "TGZ")
+# Install license file
+set ( CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/../LICENSE.md" )
+install(FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION ${INSTALL_DIR})
+set(CPACK_RPM_PACKAGE_LICENSE "MIT")
```

Comment:
The license is not set for windows before the PR. 
But here its getting set for windows as well

Reply:
Its better to have consistent license file for linux and windows.

Reply:
@mangupta - is it legally okay to have MIT license for linux and windows hip-catch-amd package ?
previously we didnt have license for windows, this PR adds the license to the package.